tracing/3401/runtime/common/src/apis.rs

```diff
@@ -115,17 +115,18 @@ macro_rules! impl_runtime_apis_plus_common {
 				}
 			}
 
+			#[cfg(not(feature = "disable-genesis-builder"))]
 			impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
 				fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
 					frame_support::genesis_builder_helper::build_state::<RuntimeGenesisConfig>(config)
 				}
 
 				fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-					frame_support::genesis_builder_helper::get_preset::<RuntimeGenesisConfig>(id, |_| None)
+					frame_support::genesis_builder_helper::get_preset::<RuntimeGenesisConfig>(id, genesis_config_preset::get_preset)
```

Comment:
All genesis should be reverted

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam-runtime-overrides/pull/125/commits/7c3d885dadc3cc2121df5c8f7f5623c5f07d2220